angelsindustries/prototypes/overrides/global-tech-base-packs.lua

```diff
@@ -78,7 +80,6 @@ if angelsmods.industries.tech then
   OV.add_prereq("angels-components-construction-3", "tech-green-packs")
   pack_replace("plastics", "orange", "green")
   pack_replace("battery", "orange", "green")
-  pack_replace("circuit-network", "orange", "green")
```

Comment:
This line didn't seem to do anything in 1.0; circuit network was green-tier by default